packages/surveys/src/types.ts

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { IItem } from "@esri/arcgis-rest-types";
+
+type IResultsAvailability = "after" | "always";
+
+export interface IFormItemProperties {
+  [propName: string]: any;
```

Comment:
Minor, concrete interface properties are generally declared  _before_ generic catch-alls like `[propName: string]: any`, similar to how we'd use the `rest` operator at the end of object or array destructuring.